src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionRoutes.scala

```diff
@@ -95,14 +95,20 @@ trait GoogleExtensionRoutes extends ExtensionRoutes with UserInfoDirectives with
                       }
                   } ~
                     pathPrefix("token") {
-                      post {
-                        entity(as[Set[String]]) { scopes =>
-                          complete {
-                            googleExtensions
-                              .getPetServiceAccountToken(WorkbenchUser(userInfo.userId, None, userInfo.userEmail, None), GoogleProject(project), scopes, samRequestContext)
-                              .map { token =>
-                                StatusCodes.OK -> JsString(token)
-                              }
+                      requireOneOfAction(
+                        FullyQualifiedResourceId(ResourceTypeName("google-project"), ResourceId("GOOGLE PROJECT")),
```

Comment:
I put these on here as hard coded strings specifically to flag them to ask for input here. There's gotta be a better way to get our resource type names than this. However, when I tried looking, there were only scattered assigning to constants across the codebase. I am curious if we have a better method by now.